src/promptflow-core/promptflow/executor/_prompty_executor.py

```diff
@@ -50,3 +50,8 @@ def _initialize_function(self):
         self._inputs = {k: v.to_flow_input_definition() for k, v in inputs.items()}
         self._is_async = False
         return self._func
+
+    def _init_input_sign(self):
```

Comment:
I think prompty also has interfaces, can we clarify whether it is not required?

Reply:
Confirmed with Clement, prompty flow also need to apply default.

Reply:
Confirmed with Clement, SDK wouldn't apply default for prompty. But I found the callable class `Prompty` will do such work in `__call__`. Consider that we should update correct inputs in run info, I prefer applying default value before run info created, update the`_input_sign` here.